apollo-model/src/main/java/io/logz/apollo/models/DeploymentHistory.java

```diff
@@ -0,0 +1,54 @@
+package io.logz.apollo.models;
+
+import java.util.List;
+
+public class DeploymentHistory {
+    public int recordsTotal;
+    public int recordsFiltered;
+    public int pageNumber;
+    public int pageSize;
```

Comment:
More indicative naming :)

Reply:
Naming was set according to API guidelines.